pkg/util/net/ip/ip_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package ip_test
```

Comment:
Shouldn't this also be set with package `ip` like the test file?

Reply:
the file was auto generated, i will verify